theano/sandbox/cuda/__init__.py

```diff
@@ -326,13 +326,10 @@ def dnn_available():
                                          " from one version, but we link with"
                                          " a different version %s" % str(v))
                     raise RuntimeError(dnn_available.msg)
-                if v == -1 or v[0] < 3007:
-                    # 3007 is the final release of cudnn v3
+                if v == -1 or v[0] < 4007:
+                    # 4007 is the final release of cudnn v4
                     dnn_available.avail = False
-                    dnn_available.msg = (
-                        "You have an old release of CuDNN (or a release "
-                        "candidate) that isn't supported.  Please update to "
-                        "at least v3 final version.")
+                    dnn_available.msg = "Version too old. Update to v5."
```

Comment:
As we have multiple dependencies, I think it's important to know the version of what is too old.
Maybe  "Your cuDNN version too old. Update to v5."